Website/src/items/content/EdgeWebsite/Data/CssClassNames/left-full.yml

```diff
@@ -0,0 +1,31 @@
+﻿---
+ID: "5f74bc88-204f-46e8-8255-7a17d6f8838a"
+Parent: "225eac13-d01e-4d05-9c41-031e5e1c234e"
+Template: "9d131d4b-98ed-4e44-a81d-144356aef00a"
+Path: "/sitecore/content/EdgeWebsite/Data/CssClassNames/left-full"
```

Comment:
As discussed, it would be nice if style items would have a `cssClasses` field where we could input multiple CSS classes like `section--dark section--dark-pattern`

We would need to be attentive on how we use these in the components though. Right now, some of these fields are used as suffixes to existing classes names. That would not work anymore.